src/java.base/share/classes/java/lang/classfile/CodeBuilder.java

```diff
@@ -611,6 +618,8 @@ default CodeBuilder conversion(TypeKind fromType, TypeKind toType) {
      * @since 23
      */
     default CodeBuilder loadConstant(ConstantDesc value) {
+//        adding null check here causes error
+//        requireNonNull(value);
```

Comment:
This method is specified as allowing `null` below.
```suggestion
```

Reply:
We are thinking about whether to allow nullable or enforce `Optional` across the ClassFile API. So this may be changed to accept an `Optional` in the future.